features/collections/icon.feature

```diff
@@ -17,8 +17,10 @@ Feature: User icons
     And I press "Update"
   Then I should see the image "alt" text "Some test description"
 
-  Scenario: Add an icon to a collection
+  Scenario: Add and remove an icon to a collection
```

Comment:
Yeah, that's definitely true -- where I get conflicted about its ~good-ness is that it will never tell us if we break deleting an icon in any other possible way, so it's giving us a false sense of security. /o\ If you'd prefer not to fight with it, that's fine -- I don't mind marking this ready to merge as it is. But if you could make an issue for improving the icon tests, that would be good, and I'll write the tests tomorrow.